api/models/people.py

```diff
@@ -131,6 +147,20 @@ def __str__(self) -> str:
         role = " (admin)" if self.is_admin else ""
         return f"{super().__str__()}{role}"
 
+    @property
+    def has_enough_groups(self) -> bool:
+        """Returns `True` if coordinator has required minimum amount of groups."""
+        if self.group_set.count() >= CoordinatorGroupLimit.MIN:
```

Comment:
Here and below: `self.group_set.count() >= CoordinatorGroupLimit.MIN` is already a bool so you can just do 
`return self.group_set.count() >= CoordinatorGroupLimit.MIN`

Reply:
of course 🤦🏻